### ISSUE TITLE
通信失敗時などの処理について

### ISSUE THREAD

**Issue by TakuyaYoshida** [NIFTYCloud-mbaas/UserCommunity#198]
オブジェクトを削除した後のロールバックに困っております。
トランザクションのような処理を実装している方がいたら、どのようにやっているかご教授頂けないでしょうか？
代替処理や、そもそも…といったお話でも構いません。
よろしくお願いいたします。

**Comment by moongift** [comment]
@TakuyaYoshida 
そうですね…NoSQLの性質上、トランザクションは難しいですね。ネットワーク上だけの問題であれば、 /batch を使うこともできますが、たぶんSDKでは提供されていないような…。

http://mb.cloud.nifty.com/doc/current/rest/datastore/objectBatch.html

**Comment by TakuyaYoshida** [comment]
moongiftさん、ありがとうございます。

たとえば１０個のデータがあって、それをまとめて削除したい際にひとつでもエラーがあった場合や
途中で通信が途切れたりした場合、皆様どうやって対処しているのかというのをお聞かせ頂ければと思っております。
